templates/admin.html

```
@@ -736,6 +736,179 @@ <h2>SMTP settings</h2>
+  <p style="color: var(--error-text); font-size: 0.875rem;">✗ {{ sms_test_detail or "The SMS gateway refused the request." }}</p>
+  {% endif %}
+
+  {% if sms_configured and not captcha_configured %}
+  <p style="color: var(--error-text); font-size: 0.85rem; border: 1px solid var(--error-text); border-radius: 6px; padding: 0.625rem 0.875rem;">
+    The booking form is public and the recipient number comes from the guest, so SMS without a captcha is an open relay someone else can bill you for. Configure the captcha above, and restrict destination countries in your gateway's own settings.
+  </p>
+  {% endif %}
+
+  {% if sms_configured %}
+  <div style="font-size: 0.85rem; color: var(--text-secondary); margin-bottom: 1rem;">
+    Sent today: <strong>{{ sms_sent_today }}</strong>{% if sms_daily_cap > 0 %} of {{ sms_daily_cap }}{% endif %}{% if sms_cost_currency %} &middot; {{ sms_cost_today }} {{ sms_cost_currency }}{% endif %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)